appveyor.yml

```diff
@@ -17,19 +17,19 @@ environment:
     ONNX_NAMESPACE: ONNX_NAMESPACE_FOO_BAR_FOR_CI
 
   matrix:
-  - CONDA_PREFIX: C:\Miniconda35-x64
-    ONNX_VERIFY_PROTO3: 1
+  #- CONDA_PREFIX: C:\Miniconda35-x64
+  #  ONNX_VERIFY_PROTO3: 1
 
-  - CONDA_PREFIX: C:\Miniconda36-x64
-    ONNX_VERIFY_PROTO3: 1
+  #- CONDA_PREFIX: C:\Miniconda36-x64
+  #  ONNX_VERIFY_PROTO3: 1
 
-  - CONDA_PREFIX: C:\Miniconda36
+  #- CONDA_PREFIX: C:\Miniconda36
 
-  - CONDA_PREFIX: C:\Miniconda35
+  #- CONDA_PREFIX: C:\Miniconda35
```

Comment:
Why are all these test configurations being removed? The CI is failing right now and the ideal fix should be to fix the CI ad not remove the failing configs?

Reply:
Sorry for that, I just wanted to have faster iterations, I forgot to uncomment them before creating the PR.

Reply:
Python 3.5 is failing with this message: *IPython 7.10+ supports Python 3.6 and above*.

Reply:
I fixed the appveyor build (force the use of IPython 7.9 for python 3.5). This fix would unblock many PR too.